openpdf/src/main/java/com/lowagie/text/Element.java

```diff
@@ -321,29 +324,30 @@ public interface Element {
      * @return    true if this is a 'content' element; false if this is a 'metadata' element
      */
 
-    boolean isContent();
+    public boolean isContent();
     
     /**
      * Checks if this element is nestable.
      * @since    iText 2.0.8
      * @return    true if this element can be nested inside other elements.
      */
 
-    boolean isNestable();
+    public boolean isNestable();
     
     /**
      * Gets all the chunks in this element.
      * 
      * @return an <CODE>ArrayList</CODE>
      */
 
-    ArrayList<Element> getChunks();
+    public ArrayList<Element> getChunks();
 
```

Comment:
Java interface is public by default, NO need to declare it as public, which makes code heavy.